File: 3136. Valid Word.java
class Solution {
    public boolean isValid(String word) {
        word = word.toLowerCase();
        int vowelCount = 0;
        int consonentCount = 0;
        for (int i = 0; i < word.length(); i++) {
            char c = word.charAt(i);
            if (!Character.isDigit(c) && (c < 'a' || c > 'z')) {
                return false;
            }
            if (!Character.isDigit(c)) {
                if (c == 'a' || c == 'e' || c == 'i' || c == 'o' || c == 'u') {
                    vowelCount++;
                } else {
                    consonentCount++;
                }
            }
        }
        return word.length() >= 3 && vowelCount >= 1 && consonentCount >= 1;
    }
}

File: 3136. Valid Word.py
class Solution:
    def isValid(self, word: str) -> bool:
        vowelCount, consonantCount = 0, 0
        for i in range(len(word)):
            c = word[i]
            if c in "aeiou" or c in "AEIOU":
                vowelCount += 1
            elif c in "bcdfghjklmnpqrstvwxyz" or c in "BCDFGHJKLMNPQRSTVWXYZ":
                consonantCount += 1
            elif c in "1234567890":
                pass
            else:
                return False
        return len(word) >= 3 and vowelCount > 0 and consonantCount > 0